File: Frontend/angular-app/src/app/modules/bursary/entities/accounting-document.ts
export class AccountingDocument {

  DocumentNo : string | undefined = undefined
  DocumentId : number | undefined = undefined
  DocumentDate : Date | undefined = undefined
  CodeVoucherGroupId : number | undefined = undefined
  DebitAccountHeadId : number | undefined = undefined
  DebitAccountReferenceGroupId : number | undefined = undefined
  DebitAccountReferenceId : number | undefined = undefined
  CreditAccountHeadId : number | undefined = undefined
  CreditAccountReferenceGroupId : number | undefined = undefined
  CreditAccountReferenceId : number | undefined = undefined
  Amount  : number | undefined = undefined
  DocumentTypeBaseId : number | undefined = undefined
  SheetUniqueNumber : string | undefined = undefined
  CurrencyFee:number|undefined = undefined;
  CurrencyAmount : number| undefined = undefined;
  CurrencyTypeBaseId : number | undefined = undefined;
  NonRialStatus : number | undefined = undefined;
  ChequeSheetId : number | undefined = undefined;
  IsRial : boolean | undefined = undefined;
  ReferenceName : string | undefined = undefined;
  ReferenceCode : string | undefined = undefined;
  Description : string | undefined = undefined;
  BesCurrencyStatus : boolean | undefined = undefined;
  BedCurrencyStatus : boolean | undefined = undefined;
}

File: Backend/Microservices/Bursary/Eefa.Bursary.Application/Models/AccountingDocument.cs
using System;

namespace Eefa.Bursary.Application.Models
{
    public class AccountingDocument
    {
        public int DocumentNo { get; set; } 
        public int DocumentId { get; set; }
        public DateTime DocumentDate { get; set; }
        public int CodeVoucherGroupId { get; set; }
        public int DebitAccountHeadId { get; set; }
        public int DebitAccountReferenceGroupId { get; set; }
        public int DebitAccountReferenceId { get; set; }
        public int CreditAccountHeadId { get; set; }
        public int CreditAccountReferenceGroupId { get; set; }
        public int CreditAccountReferenceId { get; set; }
        public decimal Amount { get; set; }
        public int DocumentTypeBaseId { get; set; }
        public int SheetUniqueNumber { get; set; }
        public int CurrencyFee { get; set; }
        public decimal CurrencyAmount { get; set; }
        public int CurrencyTypeBaseId { get; set; }
        public int NonRialStatus { get; set; }
        public int ChequeSheetId { get; set; }
        public bool IsRial { get; set; }
        public string ReferenceName { get; set; }
        public string ReferenceCode { get; set; }
        public string Description { get; set; }
    }
}
